### PR TITLE
Feature/GitHub mutation

### PR DESCRIPTION
**Pull request by BryanKAdams**
# feature to save github repos to the backend
## uses the backend branch "patchwork/GitHub mutation"

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

**Review by RobertDGordon — APPROVED** [review]
Working well here, nice job with the re-render fix.

**Review by Cireimu — APPROVED** [review]
Looks good, adding repos works quite well, also seems to fix our issues that have occurred after we pruned the files on Friday.

File: src/components/Project/index.jsx [hunk 1]
```diff
@@ -10,75 +10,86 @@ import Grade from './Grade';
...
+						<div className={teamContainer}>
+							<Team projectId={id} />
+						</div>
+					</div>
+				</div>
+			</div>
+		);
+	} else {
+		return <p>End of line.</p>;
+	}
```
Comment (lawsonarichard) [inline src/components/Project/index.jsx]: Curious to why it's formatting like this....creates extra work when we merge.

Comment (RobertDGordon) [reply src/components/Project/index.jsx]: Seems to be an issue with Prettier. I noticed this with other commits:

https://github.com/Lambda-School-Labs/mission-control-fe/pull/239/commits/3a1244bed89cc9d81ea2d46941c53a79d08935a4

Comment (BryanKAdams) [reply src/components/Project/index.jsx]: I use prettier, do we have a linter for the front end? Not sure why it looks like this but I would have to say prettier is doing it.

Comment (lawsonarichard) [reply src/components/Project/index.jsx]: Hmmm!

looks like there is a linter on front end

` "extends": ["airbnb", "plugin:prettier/recommended", "prettier/react"],`

and in .prettierrc

`{
  "printWidth": 80,
  "singleQuote": true,
  "trailingComma": "es5"
}
`

**Review by Trip4077 — APPROVED** [review]

**Review by lawsonarichard — APPROVED** [review]
Pulled down with[ needed Back end](https://github.com/Lambda-School-Labs/mission-control-be/pull/117)
Added some more repos to see if charts populate on front end on Labs 19 mission control.

Created status label of good.

Functionally seems good!

Tested on Mac OS

**Pull request merged**